db_harmonisation/crude_db_harmonisation.py

```diff
@@ -116,6 +117,20 @@ def run_rgi(fa):
 def move_mappings_to_argnorm(aro_mapping):
     shutil.copy(aro_mapping, '../argnorm/data')
 
+@TaskGenerator
+def get_rgi_hit_counts():
+    dfs = []
+
+    for dir in os.listdir('./mapping'):
+        if 'rgi.txt' in dir or 'rgi_output.txt' in dir:
+            dfs.append(pd.read_csv(f'./mapping/{dir}', sep='\t'))
+            
+    comb_df = pd.concat(dfs)
+    comb_df.to_csv('./mapping/combined_ARO_mapping.tsv', sep='\t')
+
+    for i in set(comb_df['Cut_Off']):
```

Comment:
Can you write this out to a file instead of printing it on the screen?

Or just remove these lines as they are quite a trivial computation from the tsv file saved above